docs/UserGuide.md

```diff
@@ -137,10 +137,13 @@ _Help window displayed after running the `help` command_
 
 Adds a patient to the patient list.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [d/TASK_DESCRIPTION]… [t/TAG]…`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [d/TASK_DESCRIPTION | DATE TIME]… [t/TAG]…`
+
+* Date and time should be in the format like 2022-10-16T10:15:30
+* If not date and time is provided, then a default date and time of 24 hours from creation will be set
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 d/Administer 3ml of example medicine [t/Severe]`
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 d/Administer 3ml of example medicine | 2022-10-16T10:15:30 [t/Severe]`
```

Comment:
This command does not work for some reason, it only works if the date-time part is removed.

Reply:
hmmm let me take a look

Reply:
it was a problem with the tag, is should not be in [], let me fix the documentation.